projects/packages/videopress/src/client/state/reducers.js

```diff
@@ -288,6 +281,84 @@ const videos = ( state, action ) => {
 			};
 		}
 
+		case UPLOADING_VIDEO: {
```

Comment:
`UPLOADING_VIDEO` seems to be a state more than an action. I think something like `SET_UPLOADING_VIDEO_STATE` reflexes more accurately what's happening.

Anyway, you know I'm not a suitable person to say this, but you get the idea.